lambda/action_network.py

```diff
@@ -0,0 +1,138 @@
+import logging
+import os
+from typing import Dict
+
+import requests
+from dateutil import parser
+
+log = logging.getLogger(__name__)
+
+
+def get_object_or_empty_dict(key: str, event: Dict) -> Dict:
+    location = event.get(key)
```

Comment:
don't need a function to do what `.get` does: `location = event.get(key, {})` (or if key might be set to None, `event.get(key, {}) or {}`)

Reply:
This is called 3 times.  I feel that the function call is cleaner than typing event.get(key, {}) or {}) 3 times, but this one is not worth a back-and-forth.  I'm just going to make the change.